freshmaker/config.py

```diff
@@ -92,7 +92,6 @@ def init_config(app):
     # package -> /conf/config.py.
 
     elif ('FRESHMAKER_DEVELOPER_ENV' in os.environ and
-          'FRESHMAKER_CONFIG_FILE' not in os.environ and
```

Comment:
Why removing this?

Reply:
I remember we were talking about it, If you check code a little bit further there is already that 'if' condition, and it's meaning opposite to one mentioned here. And sometimes because of that line when we are running dev scripts the wrong configuration could be used.

Reply:
Gotcha. Thank for the reminder.